src/lib/stitching/kaws/stitching.ts

```diff
@@ -147,6 +150,34 @@ export const kawsStitchingEnvironmentV2 = (
           },
         },
       },
+      HomePageMarketingCollectionsModule: {
+        results: {
+          fragment: gql`
+            ... on HomePageMarketingCollectionsModule {
+              __typename
+            }
+          `,
+          resolve: (_source, _args, context, info) => {
+            // We hard-code the collections slugs here in MP so that the app can
+            // display different collections based only on an MP change (and not
+            // an app deploy).
+            return info.mergeInfo.delegateToSchema({
+              schema: kawsSchema,
+              operation: "query",
+              fieldName: "marketingCollections",
+              args: {
+                slugs: [
```

Comment:
If it becomes more trouble to manage these via PRs, we could also consider making it an ENV var! But like the direction of putting this here vs. the app 🎉.

Reply:
An ENV var makes sense – I don't think these are expected to really change that often but if they ever need to, I think we can make the move then 👍